cli/tests/integration_tests.rs

```diff
@@ -395,38 +395,6 @@ fn bundle_single_module() {
   assert_eq!(output.stderr, b"");
 }
 
-#[test]
-fn bundle_json() {
-  let json_modules = util::root_path().join("cli/tests/020_json_modules.ts");
-  assert!(json_modules.is_file());
-  let t = TempDir::new().expect("tempdir fail");
-  let bundle = t.path().join("020_json_modules.bundle.js");
-  let mut deno = util::deno_cmd()
-    .current_dir(util::root_path())
-    .arg("bundle")
-    .arg(json_modules)
-    .arg(&bundle)
-    .spawn()
```

Comment:
@ry thanks for pointing this out; I guess there should be permission check involved in this case like a regular disk read?